www/docs/configuration/callbacks.md

```diff
@@ -25,7 +25,7 @@ You can specify a handler for any of the callbacks below.
     session: async (session, user) => {
       return Promise.resolve(session)
     },
-    jwt: async (token, user, account, profile, isNewUser) => {
+    jwt: async (token, user, account, profile, isNewUser, idToken) => {
```

Comment:
@iaincollins This (and probably any other public-facing function signature using more than 2 params) should at some point (maybe v4) become an object to leverage named parameters It is much easier for the user to pick only those params that they are going to use.